### PR TITLE
Run only networking related tests on multus and genie test lanes

### PR DESCRIPTION
**Pull request by rmohr**
**What this PR does / why we need it**:

Reduce the amount of tests to run for the networking related lanes.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by rmohr** [comment]
Hi Guys, what do you think about only running networking related tests for the clusters which bring in special networking add-ons?

**Comment by cynepco3hahue** [comment]
I thought about networking and maybe lifecycle tests,  just to check that different networking does not break our controllers.

File: automation/test.sh [hunk 1]
```diff
@@ -216,6 +216,11 @@ spec:
 EOF
   # Run only Windows tests
   ginko_params="$ginko_params --ginkgo.focus=Windows"
+
+elif [[ $TARGET =~ multus.* ]] || [[ $TARGET =~ genie.* ]]; then
+  # Run networking tests only (general networking, multus, genie, ...)
+  # If multus or genie is not present the test will  be skipped base on per-test checks
+  ginko_params="$ginko_params --ginkgo.focus=Networking"
```
Comment (yuvalif) [inline automation/test.sh]: maybe also add the "Expose" tests? make sure that services play nicely with multus and genie?
```
--ginkgo.focus=Networking|Expose"
```

**Comment by rmohr** [comment]
/hold cancel

Filtering now with `--ginkgo.focus=Networking|VMIlifecycle|Expose`. The `Networking` filter also includes genie and multus tests.

File: automation/test.sh (hunk 1 above)
Comment (rmohr) [reply automation/test.sh]: Fixed, thx.

**Comment by fabiand** [comment]
Ping?

**Pull request merged**